integration-tests/test.sh

```diff
@@ -25,17 +29,19 @@ for arch in amd64 386;do
   out=$(docker exec goss_int_test_$os bash -c "time /tmp/goss/$os/goss-linux-$arch -g /tmp/goss/$os/goss.json validate")
   echo "$out"
 
-  grep -q 'Count: 39, Failed: 0' <<<"$out"
+  egrep -q 'Count: [0-9]{2}, Failed: 0' <<<"$out"
```

Comment:
What do you want me to do? Just add a conditional for if `$os == arch`? It's a different number of tests.